src/raft_server.cxx

```diff
@@ -117,6 +118,21 @@ raft_server::raft_server(context* ctx, const init_options& opt)
     if (params->stale_log_gap_ < params->fresh_log_gap_) {
         params->stale_log_gap_ = params->fresh_log_gap_;
     }
+    if (params->enable_randomized_snapshot_creation_ && !get_last_snapshot()) {
```

Comment:
I think another condition is needed
```c++
params->snapshot_distance_ > 1
```
For 0, the snapshot is disabled, and for 1, there is no meaning of random generation.

Reply:
Maybe it is better use 0 for code readability.